cli/module_graph/analyzer.rs

```diff
@@ -0,0 +1,375 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
I'd prefer it if this file and cli/module_graph/mod.rs were merged into one file: cli/module_graph.rs

Subdirectories tend to impose a structure that is less malleable than flat organizations.

Reply:
Done